docker-compose-local.yml

```diff
@@ -56,6 +56,9 @@ services:
     environment:
       - DAR_NOTIFY_DEFAULT_URL=http://darts-stub-services:4551/VIQDARNotifyEvent/DARNotifyEvent.asmx
       - AAD_B2C_ROPC_CLIENT_ID_KEY
+      - AAD_TENANT_ID
```

Comment:
I suppose we should update the README.md to cover these `Environment variables` that we need in the application, like in darts-api?

Reply:
Done